cosinnus_event/migrations/0022_auto_20220210_1248.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2 on 2022-02-10 11:48
+
+import cosinnus_event.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cosinnus_event', '0021_auto_20211022_1029'),
+    ]
+
+    operations = [
```

Comment:
Please add a data migration step in here to migrate the previous values of 'is_confidential' for events that had already had them set. Otherwise, we would lose the data for existing events.

You can do this by re-ordering and amending the operations so that in order:

1. the two new fields are added
2. a data migration is exectued that converts and saves the value of the previous field appropriately into the new fields (I'd imagine something like `event.is_public = not event.is_confidential; event.is_visible = not event.is_confidential`. This data migration should have a `reverse` operation that implements the inverted value assignments!
3. the old field 'is_confidential' is removed